src/App.tsx

```diff
@@ -95,7 +111,7 @@ async function login() {
 async function logout() {
   console.log('Logging out');
 
-  await loginClient.logout();
+  await loginClient.logout('http://localhost:8080');
```

Comment:
I think sticking with the default is going to make more sense `window.location.href`, or at the very least should likely be `window.location.origin` so it doesn't immediately break if logout is deployed in production.

```suggestion
  await loginClient.logout(window.location.href);
```